data-intake/ddi/extension-table/jupyter/name-rank-sample.ipynb

```diff
@@ -16,17 +16,15 @@
     "\n",
     "### Prerequisites\n",
     "Before running this sample, do the following:\n",
-    "1. In your Visier tenant, create a simple numeric property with the object name `Name_Rank` on the `Employee` object in Visier. Publish the property to production.\n",
+    "1. In your Visier tenant, create a simple number property with the object name `Name_Rank` on the `Employee` object in Visier. Publish the property to production.\n",
```

Comment:
why change this? Emily reviewed all the strings already on this page.

Reply:
We don' have `numeric` property, and it could confuse while creating property.

Reply:
If the property MUST be data type "Number", this change is fine. If the property can be any numeric data type, eg integer, number, or others, we should use the generic "numeric".

Reply:
Any numeric will do. 'simple number property' isn't great grammatically. I'd say leave it as it was

Reply:
Okay, I rolled back this change.